src/styles/utils/_variables.scss

```
@@ -0,0 +1,7 @@
+$card-height: 300px;
+
+$media-queries:
+  "1px" 1,
+  "488px" 2,
+  "768px" 3,
+  "1024px" 4;
```

No need to specify (min-width: 1px) in $media-queries, you've already created grid with 1 column and in media query you should change amount of columns
```suggestion
$media-queries:
  "488px" 2,
  "768px" 3,
  "1024px" 4;
```

---

**FIxed**